app/models/procedure.rb

```diff
@@ -74,7 +74,7 @@ def active_revision
     brouillon? ? draft_revision : published_revision
   end
 
-  def types_de_champ_for_procedure_presentation(parent = nil)
+  def aggregated_types_de_champ(parent = nil)
```

Comment:
```suggestion
  def all_versions_types_de_champ(parent: nil)
```

renommage puis passage en paramètre nommé ?